source/base_interface.cc

```diff
@@ -207,6 +208,7 @@ void
 BaseInterface<dim,spacedim,LAC>::
 compute_system_operators(const std::vector<shared_ptr<LATrilinos::BlockMatrix> >,
                          LinearOperator<LATrilinos::VectorType> &,
+                         LinearOperator<LATrilinos::VectorType> &,
                          LinearOperator<LATrilinos::VectorType> &) const
```

Comment:
Avevo provato a mettere come preconditioner_finer di default un NULL (tanto per mettere un valore di dafault) e poi sfruttare questo valore per usare un precondizionatore o due ma non riesco a farlo andare.
Sono aperto a nuove proposte! :)
Tuttavia credo che la possibilità dello switch possa ridurre i tempi di risoluzione di un problema, soprattutto per problemi con il flow past a cylinder dove AMG è essenziale in certi punti dove la soluzione non riesce a ripartire ma molto dispendioso dove la soluzione viene calcolata agevolmente...